src/component/Navbar.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link, NavLink } from 'react-router-dom';
+import '../index.css';
+
+const Navbar = ({ title, routes }) => (
+  <nav className="nav-bar">
+    <Link className="nav-brand" to="/">{title}</Link>
+    <ul className="nav-links">
+      {routes.map(({ name, path }) => (
+        <li key={path}>
+          <NavLink className="nav-link" activeClassName="active-link" exact to={path}>{name}</NavLink>
```

Comment:
- [ ] The Navlinks on zeplin don't have an underline while your has. Kindly modify yours so that it matches Zeplin design and use the correct font weight and variant(i.e uppercase)

your design 👇🏽 
![image](https://github.com/theresetuyi/React-Bookstore/assets/63473584/af704fd6-780c-47d1-af38-251b6eb4550c)
Zeplin design 👇🏽 
![image](https://github.com/theresetuyi/React-Bookstore/assets/63473584/7544e3e0-77cc-4d8c-a53a-f32b4231fa96)